elmslie-core/src/main/java/vivid/money/elmslie/core/store/ElmStore.kt

```diff
@@ -25,6 +26,7 @@ class ElmStore<Event : Any, State : Any, Effect : Any, Command : Any>(
 ) : Store<Event, Effect, State> {
 
     private val logger = ElmslieConfig.logger
+    private val eventScope = ElmEventScope("EventScope")
```

Comment:
I now that all these properties are sorted by modifiers visibility, but what if we group scopes together?)

Reply:
ok